### ISSUE TITLE
Bug: Nondescript Max-Filesize Error when uploading from the Asset or Image Inspector

### ISSUE THREAD

**Issue by marcrobertscamao** [neos/neos-ui#3398]
### Description

When uploading an asset or image over the inspector using the file upload icon or drag and drop the following nondescript and confusing error message for the editor is displayed: 

"Validation failed while trying to call Neos\Neos\Controller\Backend\ContentController->uploadAssetAction()."

Performing the same action via the media view returns the more helpful error message:

"The file size of 17.05 MB exceeds the allowed limit of 8.00 MB for the file "Big_File.jpg"

### Steps to Reproduce

1. Select an element with an asset or image inspector
2. Click the upload icon
3. Select the large file in the open file inspector
4. Click Open to trigger the upload

#### Expected behavior

You should see a friendly error message as in the media view "The file size of 17.05 MB exceeds the allowed limit of 8.00 MB for the file "Big_File.jpg"

#### Actual behavior

The toast notification is shown with the error message "Validation failed while trying to call Neos\Neos\Controller\Backend\ContentController->uploadAssetAction()."

### Affected Versions

Neos: 8.2.2

UI: v8.1.6
